integration/heapster_api_test.go

```diff
@@ -1001,6 +1035,10 @@ func apiTest(kubeVersion string, zone string) error {
 	if err := buildAndPushDockerImages(fm, zone); err != nil {
 		return err
 	}
+
+	setupHeapsterServiceAccount(fm)
```

Comment:
You will have to call setupHeapsterServiceAccount and setupHeapsterRBAC after the test namespace is created.

Reply:
Will do.